src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -160,6 +160,13 @@ public new OpenIdConnectEvents Events
         }
 
         /// <summary>
+        /// Gets or sets the 'max_age'. If the <see cref="TimeSpan"/> is at least one second, the 'max_age' parameter will be sent with the
+        /// authentication request. If the identity provider has not actively authenticated the user within the length of time specified, the
+        /// user will be prompted to re-authenticate. By default no max_age is specified.
+        /// </summary>
+        public TimeSpan MaxAge { get; set; } = new TimeSpan(0, 0, 0);
```

Comment:
It would be preferable to use a nullable property so the `max_age=0` value can be validly represented.